content/tagging/assigning_tags.md

```diff
@@ -13,25 +13,195 @@ further_reading:
 
-The following [integrations][1] sources create tags automatically in Datadog:
+In YAML files, use a tag dictionary with a list of tags you want assigned at that level. Any tag you assign to the Agent is applied to every integration on that Agent's host.
 
-|                                         |                                                                                                                                                                                                                                                                                                                                               |
-| :-------------------------------------  | :------------------------------------------------------------------------------                                                                                                                                                                                                                                                               |
+Tag dictionaries have two different yet functionally equivalent forms:
+
+```
+tags: key_first_tag:value_1, key_second_tag:value_2, key_third_tag:value_3
+```
+
+or
+
+```
+tags:
+    - key_first_tag:value_1
+    - key_second_tag:value_2
+    - key_third_tag:value_3
+```
+
+It is recommended you assign tags as `key:value` pairs but simple tags are also accepted. See [how to use][61] tagging for more details.
+
+### Environment Variables
+
+When installing the containerized Datadog Agent, host tags can be set using the environment variable `DD_TAGS`. We automatically collect common tags from [Docker][77], [Kubernetes][78], [ECS][79], [Swarm, Mesos, Nomad and Rancher][77]. To extract even more tags, use the following options:
+
+| Environment Variable               | Description                                    |
+|------------------------------------|------------------------------------------------|
+| `DD_DOCKER_LABELS_AS_TAGS`         | extract docker container labels                |
+| `DD_DOCKER_ENV_AS_TAGS`            | extract docker container environment variables |
+| `DD_KUBERNETES_POD_LABELS_AS_TAGS` | extract pod labels                             |
+
+**Examples:**
+
+```shell
+DD_KUBERNETES_POD_LABELS_AS_TAGS='{"app":"kube_app","release":"helm_release"}'
+DD_DOCKER_LABELS_AS_TAGS='{"com.docker.compose.service":"service_name"}'
+```
+
+Either define the variables in your custom `datadog.yaml`, or set them as JSON maps in these envvars. The map key is the source (`label/envvar`) name, and the map value is the Datadog tag name.
+
+### UI
+
+{{< tabs >}}
+{{% tab "Infrastructure List" %}}
+
+You can assign host tags in the UI via the [Infrastructure List][62]. Click on any host to show the host overlay on the right. Then under the *User* section, click the **Edit Tags** button. Enter the tags as a comma separated list then click **Save Tags**.
```

Comment:
when you give a spatial indication, you should set the reference point. 
r/`on the right`/ `on the right of the page`